main.js

```diff
@@ -158,7 +158,16 @@ $(function() {
         let storage = getStorage();
 
         storage.forEach(function(item) {
-          document.querySelector('#LC' + item.key.split('-')[0] + ' .collapser').click();
+          let [lower, upper] = item.key.split('-');
+          upper++;
+          let lowerLine = document.querySelector('#LC' + lower).textContent;
+          let upperLine = document.querySelector('#LC' + upper).textContent;
+
+          if (lowerLine.includes('{') && upperLine.includes('}')) {
```

Comment:
I'm not really sure how to adapt your whitespace counting code into what I'm trying to do. Any ideas how I might use it to test if anything changed?